### ISSUE TITLE
Closing dialogs using Esc doesn't work anymore on Jetbrains

### ISSUE THREAD

**Issue by tbressers** [continuedev/continue#2521]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS:Windows
- Continue version:0.0.75
- IDE version: JetBrains IntelliJ 2024.2.1 (Community Edition)
- Model:Llama 3.1
```

### Description

Closing some (e.g., rename file) dialogs using Esc doesn't work anymore 

### To reproduce

1. Rename a file in your project in the file overview on your left-hand side
2. Try to close the dialog using Esc(ape)

**Comment by sestinj** [comment]
@Patrick-Erichsen it seems likely that this is coming from the input box. I know you're working on some polish there this week, thought I'd just make you aware of this

**Comment by Patrick-Erichsen** [comment]
Yes, will be taking a look at this ASAP, thanks for the heads up @tbressers !

**Comment by Patrick-Erichsen** [comment]
@tbressers should be fixed here: https://github.com/continuedev/continue/pull/2624

We'll be cutting a new EAP release by end of day, but if you're not on EAP, we are cutting a new main release by end of week.

Thanks again for bringing this to our attention!
